ouroboros-network-framework/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 
 * Split `test` component into `io-tests` and `sim-tests`.
 
+- Fix MultiNodeScript Generator, in particular fix the delays between two events that
```

Comment:
```suggestion
* Fix MultiNodeScript Generator, in particular fix the delays between two events that
```

I have an impression that your editor is configured to use `-`, but we have `*` in `CHANGELOG`s.